### DISCUSSION TITLE
32 bit compatibility

### DISCUSSION THREAD

**Discussion by Nacsis** [Darkside138/DiscordSoundboard#166] [Q&A] [answered]
Hi,
I used to run the bot on my Raspberry Pi 4. As far as I see the soundboard only works with 64bit java. However it is hard to get 64bit java running on the raspberry pi as it does not come with 64bit support.

Is there a 32bit version of the soundboard or a way to make the current version compatible with 32bit java?

Kind regards

**Comment by Darkside138** [comment, marked as answer]
Have you tried the latest version? I think it should work on 32bit but I'm not 100% sure
